web/next/src/app/page.tsx

```
@@ -123,57 +163,36 @@ bun dev`
                 >
-                  {item}
-                </span>
-              ))}
+                  <Icon className="h-5 w-5 shrink-0" />
+                  <span className="text-sm font-medium">{tech.name}</span>
+                </div>
+              )
+            })}
           </div>
           <style
             dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)